openmc/geometry.py

```diff
@@ -257,7 +257,49 @@ def get_all_lattices(self):
                     lattices[cell.fill.id] = cell.fill
 
         return lattices
+    
+    def get_all_surfaces(self):
+        """
+        Return all surfaces used in the geometry
+
+        Returns
+        -------
+        collections.OrderedDict
+            Dictionary mapping surface IDs to :class:`openmc.Surface` instances
 
+        """
+        surfaces = OrderedDict()
+        
+        for cell in self.get_all_cells().values():
+            self._get_surfaces_from_region(surfaces, cell.region)
+        return surfaces
+    
+    def _get_surfaces_from_region(self, surfaces, region):
```

Comment:
On second thought, why not move method to the `Region` class? It would recurse through all of the nodes and return a `dict` of the `Surface` instances keyed by their IDs. This `dict` would simply be added to the global `dict` instantiated for the entire `Geometry`. Not only would this be cleaner (IMHO), it might also be nice to retrieve all of the `Surface` instances from a `Region`, in some cases.